src/App.tsx

```diff
@@ -1,18 +1,33 @@
-import React from 'react'
+import React, { createContext, useReducer } from 'react'
```

Comment:
`{ createContext, useReducer }` is unused
```suggestion
import React from 'react'
```